app/policies/assignment_policy.rb

```diff
@@ -53,4 +51,28 @@ def not_yet_in_group?
   def autogenerate_group_name?
     record.group_name_autogenerated
   end
+
+  def manage?
+    allowed_to?(:manage_assessments?, with: GraderPermissionPolicy)
+  end
+
+  def can_run_tests?
+    allowed_to?(:grader_run_tests?, with: GraderPermissionPolicy) || user.student?
+  end
+
+  def view?
```

Comment:
I liked the name you gave in `GradeEntryFormPolicy`, `grading?`. Please use that here as well.